2023/01/26 22:48:00 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/v1/hello HTTP/1.1", upstream: "http://127.0.0.1:8080/api/v1/hello", host: "127.0.0.1"
2023/01/26 22:48:00 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/v1/hello HTTP/1.1", upstream: "http://127.0.0.1:8080/api/v1/hello", host: "127.0.0.1"
2023/01/26 22:54:53 [error] 22#22: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/v1/hello HTTP/1.1", upstream: "http://127.0.0.1:8080/api/v1/hello", host: "127.0.0.1"
2023/01/26 22:54:53 [error] 22#22: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/v1/hello HTTP/1.1", upstream: "http://127.0.0.1:8080/api/v1/hello", host: "127.0.0.1"
2023/01/26 22:55:31 [error] 22#22: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/v1/hello HTTP/1.1", upstream: "http://127.0.0.1:8080/api/v1/hello", host: "127.0.0.1"
2023/01/26 22:55:31 [error] 22#22: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/v1/hello HTTP/1.1", upstream: "http://127.0.0.1:8080/api/v1/hello", host: "127.0.0.1"
